src/Network/CorsBuilder.php

```diff
@@ -14,6 +14,8 @@
  */
 namespace Cake\Network;
 
+use Cake\Http\Response;
```

Comment:
I don't think the extra alias helps much. Using the old alias is ok here to me at least.